OrbitGl/CaptureWindow.cpp

```diff
@@ -30,6 +31,8 @@
 #include "LinuxUtils.h"
 #endif
 
+ABSL_DECLARE_FLAG(bool, non_release_features);
```

Comment:
I would suggest to name this "experimental_features" to avoid the negation, which is always harder to understand.